cli/plasmo/src/features/extension-devtools/project-path.ts

```diff
@@ -31,20 +33,19 @@ const getWatchReasonMap = (paths: string[], reason: WatchReason) =>
     return output
   }, {}) as Record<string, WatchReason>
 
-const getIndexList = (projectDir: string, moduleName: string, ext = "tsx") => [
-  resolve(projectDir, `${moduleName}.${ext}`),
-  resolve(projectDir, moduleName, `index.${ext}`)
+const getIndexList = (projectDir: string, moduleName: string, ext = ".ts") => [
```

Comment:
Is there a time where we don't want there to be a `.` between the file name and an extension?

Reply:
This is largely to work with the other system xd... I thought about it and atm, I think we can just have it that an ext has a `.` prefix.